crypto/rsa/rsa_ossl.c

```diff
@@ -569,6 +571,8 @@ static int rsa_ossl_public_decrypt(int flen, const unsigned char *from,
             goto err;
 
     i = BN_bn2binpad(ret, buf, num);
+    if (i <= 0)
```

Comment:
the error return from BN_bn2binpad is -1
so `if (i < 0)` would look more correct.
note that the RSA_paddin_check functions happen to have a check for this error:
https://github.com/openssl/openssl/blob/03e5668343078b963cc6544ad7270743de13e514/crypto/rsa/rsa_pk1.c#L164-L165
So I would think that might indicate that the RSA_NO_PADDING really misses that.

Reply:
I do not know exactly what coverity complained about here.
I assume that it was only memcpy(..., -1) right?
a memcpy of 0 bytes would be okay, for them, or not?
I would keep the fix as small as possible, but either way it is okay.

Reply:
Yeah, it was the possible negative value being passed into memcpy.
I've switched over to using `< 0`.